python-threatexchange/threatexchange/exchanges/clients/techagainstterrorism/tests/test_api.py

```diff
@@ -0,0 +1,94 @@
+import typing as t
```

Comment:
nit: I git an internal nagger that forces me to add 

`# Copyright (c) Meta Platforms, Inc. and affiliates.` to the tops of all files, though I can do it as a followup